tests/jballerina-unit-test/src/test/resources/testng.xml

```diff
@@ -88,6 +88,36 @@
             <class name="org.ballerinalang.test.expressions.typeof.TypeofOverLiteralExpressionTest" />
             <class name="org.ballerinalang.test.expressions.binaryoperations.BinaryExprTest" />
             <class name="org.ballerinalang.test.worker.WorkerCallingFunction"/>
+
+            <!--Built in methods-->
+            <class name="org.ballerinalang.test.expressions.builtinoperations.LengthOperationTest"/>
+            <class name="org.ballerinalang.test.expressions.builtinoperations.BuiltinOperationsTest"/>
+
+            <class name="org.ballerinalang.test.expressions.conversion.NativeConversionWithStampTypesTest"/>
+
+            <class name="org.ballerinalang.test.expressions.stamp.XMLStampInbuiltFunctionTest"/>
+            <class name="org.ballerinalang.test.expressions.stamp.AnydataStampInbuiltFunctionTest"/>
+            <class name="org.ballerinalang.test.expressions.stamp.ArrayStampInbuiltFunctionTest"/>
+            <class name="org.ballerinalang.test.expressions.stamp.JSONStampInbuiltFunctionTest"/>
+            <class name="org.ballerinalang.test.expressions.stamp.MapStampInbuiltFunctionTest">
+                <!-- Fails because of cyclic record type issue-->
+                <methods>
+                    <exclude name="testStampRecordToMapWithCyclicValueReferences"/>
+                    <exclude name="testStampRecordToRecordWithCyclicValueReferences"/>
+                </methods>
+            </class>
+
+            <class name="org.ballerinalang.test.expressions.stamp.RecordStampInbuiltFunctionTest">
+                <!-- Fails because of optional fields not implemented-->
+                <methods>
+                    <exclude name="testStampRecordToRecordWithOptionalFields"/>
+                </methods>
+            </class>
+            <class name="org.ballerinalang.test.expressions.stamp.StampInbuiltFunctionNegativeTest"/>
+            <class name="org.ballerinalang.test.expressions.stamp.TupleTypeStampInbuiltFunctionTest"/>
+            <class name="org.ballerinalang.test.expressions.stamp.UnionTypeStampInbuiltFunctionTest"/>
+            <class name="org.ballerinalang.test.expressions.stamp.XMLStampInbuiltFunctionTest"/>
+       
```

Comment:
Clone , Freeze and some native conversion unit test classes are not supported yet due to compilation issues as they need some missing operators and BLangFunctionVarRef , BLangStructFunctionVarRef expressions and decimal support.